client/src/pages/CommunityPage/CommunityPage.js

```diff
@@ -151,7 +151,7 @@ const CommunityPage = props => {
         <a className="seeMore-Button">See More</a>
         <div>
           <header className="CommunityPage__header">
-            Groups in WASHINGTON, DC: <a href="./view-group"> View Sample Page </a>
+            Groups in WASHINGTON, DC: <a href="./view-group"> View Sample Page </a><a href="./create-group">Create Group</a>
```

Comment:
You can remove the sample page link, since the group pages should now link to groups!